waypoints:
  # cabinet: [5.44, 1.76, 0] # (x,y,yaw)
  # door_to_lab: [5, 6, 0]
  # white_board: [9.981, 10.514, 3.14]
  # kitchen: [1.17, 2.37, 1.76]
  # garbagecan: [1.90, 2.00, -0.09]
  # diningtable: [1.59, 3.56, -0.00]
  # fridge: [0.15, 2.99, 1.44]
  # countertop: [0.35, 2.96, 2.49]
  # sink: [0.29, 2.76, 3.12]
  # bedroom: [1.52, -3.92, -1.61]
  # bed: [1.44, -3.47, 0.24]
  # tvstand: [1.62, -4.22, 0.14]
  # couch: [0.64, -4.46, -2.09]
  # desk: [0.81, -3.97, -3.09]
  # livingroom: [1.38, -0.56, -0.00]

  kitchen: [1.10, 2.31, 1.56]
  garbagecan: [2.04, 1.91, 0.04]
  diningtable: [1.34, 3.15, 0.2]
  fridge: [0.28, 2.33, 1.67]
  countertop: [0.41, 2.99, -3.13]
  sink: [0.24, 1.97, -3.04]
  bedroom: [1.26, -4.01, -1.63]
  bed: [1.52, -3.86, 0.77]
  tvstand: [1.86, -4.58, 0.54]
  couch: [0.58, -4.48, -2.05]
  desk: [0.72, -4.04, 3.07]
  livingroom: [1.34, -0.56, -0.02]

default_frame: map
